zanata-adapter-xliff/src/main/java/org/zanata/adapter/xliff/XliffWriter.java

```diff
@@ -216,30 +221,12 @@ public static void writeFile(File file, Resource doc, String locale,
             // end xliff tag
             writer.writeEndDocument();
             writer.flush();
+            writer.close();
```

Comment:
the writer is not Closeable. I need to close the writer so that it can call the close method on its receiver (check its implementation). It's not a resource to close or clean up.